frontend/theme/index.ts

```diff
@@ -0,0 +1,103 @@
+import { fromEntries } from '../utils'
+
+export const breakpoints = {
+  XS: 600,
+  SM: 786,
+  MD: 1024,
+  LG: 1280,
+} as const
```

Comment:
Our current breakpoints are 375, 960, 1040, and 1364, but I think we can tweak these a little bit to handle more use cases. For example, we may never need 375, especially since that's the smallest value we support and if we write "mobile first" CSS, we won't need it as a breakpoint. For example:

```css
.my-box {
  /* apply to every breakpoint, starting from 0px */
  color: tomato;
  @media (min-width: 800px) {
    /* override and apply to 800px and up */
    color: papayawhip;
  }
}
```

I also think that since our breakpoints of 960 and 1040 are really close together (only 80px apart), there will rarely be situations where we want to use both. How do you feel about the breakpoints I listed above?

600 allows us to target from 0 -> 600 (all phones) and then start switching to "tablet" layouts, if necessary
768 allows us to target tablets specifically (iPad having a width of 768)
1024 is a good middle ground between our current 960 and 1040 that is still divisible by 16
1280 is another pixel value that's good for grids and can be a starting point for "desktop and beyond"